plugins/chat/spec/services/chat/update_user_last_read_spec.rb

```diff
@@ -109,6 +109,11 @@
         it "publishes new last read to clients" do
           expect(messages.map(&:channel)).to include("/chat/user-tracking-state/#{current_user.id}")
         end
+
+        it "updates the channel membership last_viewed_at datetime" do
+          membership.update!(last_viewed_at: 1.day.ago)
+          expect { result }.to change { membership.reload.last_viewed_at }
+        end
```

Comment:
This test seems weird... we check that doing update! does and update?

Reply:
I'll rewrite this and the other to compare the `last_viewed_at` directly :+1: